src/components/QuestionView/Collapsed.tsx

```diff
@@ -54,9 +48,12 @@ export default function CollapsedView({
           <ActionIcon variant="transparent" title="Edit" onClick={setActiveQuestion}>
             <Icon name="pencil" width={22} />
           </ActionIcon>
-          <ActionIcon variant="transparent">
+          <ActionIcon variant="transparent" onClick={() => setExpandedQuestionId(question.questionId)}>
             <Icon name="caretDown" />
           </ActionIcon>
+          <ActionIcon variant="transparent" className="questionHandle">
```

Comment:
Does it show up always? Can you add screenshots in the PR description?

Reply:
fixed